PWGHF/Core/HFSelectorCuts.h

```diff
@@ -398,18 +398,22 @@ constexpr double pTBins[npTBins + 1] = {
   5.,
   6.,
   8.,
-  12.};
+  12.,
+  24.,
+  36.};
 auto pTBins_v = std::vector<double>{pTBins, pTBins + npTBins + 1};
 
 // default values for the cuts
-constexpr double cuts[npTBins][nCutVars] = {{0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 0.8},  /* 0  < pT < 1  */
-                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 0.8},  /* 1  < pT < 2  */
-                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 0.8},  /* 2  < pT < 3  */
-                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 0.8},  /* 3  < pT < 4  */
-                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 0.8},  /* 4  < pT < 5  */
-                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 0.8},  /* 5  < pT < 6  */
-                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 0.8},  /* 6  < pT < 8  */
-                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 0.8}}; /* 8  < pT < 12 */
+constexpr double cuts[npTBins][nCutVars] = {{0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 50.0, 50.0, 0.8, 0.8},  /* 0  < pT < 1  */
+                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 50.0, 50.0, 0.8, 0.8},  /* 1  < pT < 2  */
+                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 50.0, 50.0, 0.8, 0.8},  /* 2  < pT < 3  */
+                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 50.0, 50.0, 0.8, 0.8},  /* 3  < pT < 4  */
+                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 50.0, 50.0, 0.8, 0.8},  /* 4  < pT < 5  */
+                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 50.0, 50.0, 0.8, 0.8},  /* 5  < pT < 6  */
+                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 50.0, 50.0, 0.8, 0.8},  /* 6  < pT < 8  */
+                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 50.0, 50.0, 0.8, 0.8},  /* 8  < pT < 12  */
+                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 50.0, 50.0, 0.8, 0.8},  /* 12  < pT < 24 */
+                                            {0.400, 0.5, 0.2, 10.0, 10.0, 1.e-3, 0.0, 50.0, 50.0, 0.8, 0.8}}; /* 24  < pT < 36 */
```

Comment:
The comment alignment is shifted here.

Reply:
Done

Reply:
It's still one space off.

Reply:
Sorry but the I do not understand where I should add a space

Reply:
The text is shifted by an extra space in the last two lines. Detail, I know, it just looks better aligned. :-)
```c++
/* 8  < pT < 12  */
/* 12  < pT < 24 */
/* 24  < pT < 36 */
      ^        ^
```

Reply:
Done